### ISSUE TITLE
BUG: docker build crashes

### ISSUE THREAD

**Issue by JWDebler** [ccdmb/predector#33]
**Describe the bug**
Using the install instructions for docker crashes

**To Reproduce**
```
 curl -s "https://raw.githubusercontent.com/ccdmb/predector/master/install.sh" | bash -s docker -3 signalp-3.0.Linux.tar.Z     -4 signalp-4.1g.Linux.tar.gz     -5 signalp-5.0b.Linux.tar.gz     -t targetp-2.0.Linux.tar.gz     -d deeploc-1.0.All.tar.gz     -m tmhmm-2.0c.Linux.tar.gz  -p phobius101_linux.tar.gz
```
**Screenshots or copy-pasted terminal outut **
```
## Setting up docker image.

main: line 532: : command not found
(23) Failed writing body

ERROR: creating the docker image failed
```

**Operating system (please enter the following information as appropriate):**
 - OS/Linux distribution: Ubuntu 18.04.4 LTS
 - Dependency management: Docker
 - Nimbus VM

**Comment by darcyabjones** [comment]
Thanks Johannes,

Can you please try running the command with the debug flag?
And copy and paste the last few lines of what's in the terminal here?
E.g.

```
 curl -s "https://raw.githubusercontent.com/ccdmb/predector/0.1.0-dev/install.sh" | bash -s docker --debug -3 signalp-3.0.Linux.tar.Z     -4 signalp-4.1g.Linux.tar.gz     -5 signalp-5.0b.Linux.tar.gz     -t targetp-2.0.Linux.tar.gz     -d deeploc-1.0.All.tar.gz     -m tmhmm-2.0c.Linux.tar.gz  -p phobius101_linux.tar.gz
```

**Comment by darcyabjones** [comment]
Fixed with  dd9b756

Use the "master" branch url until we tag a new version (> 0.1.0-dev), otherwise the fixes won't be incorporated.
E.g.

```
curl -s "https://raw.githubusercontent.com/ccdmb/predector/master/install.sh" | bash -s docker -3 signalp-3.0.Linux.tar.Z     -4 signalp-4.1g.Linux.tar.gz     -5 signalp-5.0b.Linux.tar.gz     -t targetp-2.0.Linux.tar.gz     -d deeploc-1.0.All.tar.gz     -m tmhmm-2.0c.Linux.tar.gz  -p phobius101_linux.tar.gz
```
